src/fuzzer.rs

```diff
@@ -164,6 +164,31 @@ pub fn fuzz() -> Result<()> {
     let power = StdPowerMutationalStage::new(mutator_openapi);
     let mut stages = tuple_list!(calibration, power);
 
+    // APIs already create code coverage during boot. We check if the code coverage is non-zero. Zero coverage might indicate an issue with the coverage agent or a target that was not rebooted between fuzzing runs.
+    if !config
+        .coverage_configuration
+        .equivalent(&crate::configuration::CoverageConfiguration::Endpoint)
```

Comment:
Why the IndexMap trait? There's not a map in sight right?

```suggestion
    if config.coverage_configuration != crate::configuration::CoverageConfiguration::Endpoint
```

Reply:
This is better. Thanks.